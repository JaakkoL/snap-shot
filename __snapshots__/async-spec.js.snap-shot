module.exports[`promise to named function 1`] = 20

module.exports[`snapshot can wrap promise 1`] = "promise resolved"

