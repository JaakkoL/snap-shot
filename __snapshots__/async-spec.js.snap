module.exports[`promise to named function 0`] = 20

module.exports[`snapshot can wrap promise 0`] = "promise resolved"

